docs/users/cli-usage.md

```diff
@@ -374,15 +375,23 @@ Commands can also be executed using the shorter `ddev . <cmd>` alias.
 The `ddev ssh` command will open an interactive bash shell session to the container for a ddev service. The web service is connected to by default. The session can be ended by typing `exit`. To connect to another service, use the `--service` flag to specify the service you want to connect to. For example, to connect to the database container, you would run `ddev ssh --service db`.
 
 ### Log Access
+
 The `ddev logs` command allows you to easily retrieve error logs from the web server. To follow the web server error log (watch the lines in real time), run `ddev logs -f`. When you are done, press CTRL+C to exit from the log trail.
 
-Additional logging can be accessed by using `ddev ssh` to manually review the log files you are after. The web server stores access logs at `/var/log/nginx/access.log`, and PHP-FPM logs at `/var/log/php7.0-fpm.log`.
+To manually review additional log files use the `ddev ssh` command. The web server stores access logs at `/var/log/nginx/access.log`, and PHP-FPM logs at `/var/log/php7.0-fpm.log`.
 
 ## Stopping a project
-You can stop a project's containers without losing data by using `ddev stop` in the working directory of the project. You can also stop any running project's containers by providing the project name as an argument, e.g. `ddev stop <projectname>`.
+
+Stop a project's containers without losing data by using `ddev stop` in the working directory of the project. Stop any running project's containers by providing the project name as an argument, e.g. `ddev stop <projectname>`.
```

Comment:
This is quite old and incorrect, thought it had been fixed. Neither `ddev stop` nor `ddev rm` (without --remove-data) cause any loss of data. I'd recommend just removing this stanza, as there's not really a use for `ddev stop` at this point, and we're working on moving to rm, but perhaps with a rename of the command.

Reply:
Taking out this whole thing now 

``
## Stopping a project

Stop a project's containers without losing data by using `ddev stop` in the working directory of the project. Stop any running project's containers by providing the project name as an argument, e.g. `ddev stop <projectname>`.
``